fortuna/src/casus/blocks/BoundingBox.js

```diff
@@ -0,0 +1,19 @@
+//@flow strict
+
```

Comment:
Okay, this stuff is going to cause all sorts of merge conflicts when you merge it I think. You might have to undo some of the commits merging stuff in that you didn't write so we don't have two copies everywhere